ledger/simulation/tracer.go

```diff
@@ -245,6 +251,32 @@ func (tracer *evalTracer) makeOpcodeTraceUnit(cx *logic.EvalContext) OpcodeTrace
 	return OpcodeTraceUnit{PC: uint64(cx.PC())}
 }
 
+func (o *OpcodeTraceUnit) appendDeletedStackValue(cx *logic.EvalContext, tracer *evalTracer) {
+	tracer.stackChangeExplanation = cx.NextStackChange()
+
+	stackCopy := cx.Stack()
```

Comment:
I think it's slightly confusing to call this `stackCopy`. It makes it seem like modifying it is ok. But it isn't - it would change the stack in the real `cx`.

Reply:
I wanted to avoid the (potential) copy, so I capitalized `stack` to `Stack` from `cx *EvalContext`.